packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -312,6 +312,23 @@ export class FASTHorizontalScroll extends FASTElement {
         this.setFlippers();
     }
 
+    /**
+     * Checks to see if the stops are returning values
+     *  otherwise it will try to reinitialize them
+     *
+     * @returns boolean for the current state of scrollStops are non-zero values
+     * @public
+     */
+    public validateStops(reinit: boolean = true): boolean {
```

Comment:
Can't this be a private function?

Reply:
Everything is private in horizontal-scroll and it's being inherited and used everywhere making overrides impossible. I thought there might be a use case where this might need to be called in to. Perhaps lazy loaded content or expanding content. But they are probably not going to be a problem and if they are, this probably won't help. Updated to private.